pages/glossary/changes-in-content.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Changes in content
+key: changes-in-content
+unambiguous: true
+objective: true
+input_aspects:
+  - Accessibility tree
+  - Audio output
+  - CSS styling
+  - DOM tree
+---
+
+A _change in the content_ of a [web page][] caused by an [event][] occurs when, by comparing the [web page][] before and 1 minute after the event [firing][], at least one of the following occurs:
+
+- **visible changes:** the rendered pixels change in any part of the document that is currently within the [viewport][] or that can be brought into the [viewport][] via scrolling; or
+- **accessibility tree changes:** any state, property or event of a node representing an [accessible object][] of the [accessibility tree][] changes, or any node is inserted in, or removed from the [accessibility tree][]; or
+- **audible changes:** the audio rendered by the [web page][] changes.
+
+- If the [web page][] is rendering time-based media, rendered pixels and audio will be changing as part of the playback. The comparison in this instance should compare the pixels and audio that are rendered if the event is not fired, with the ones that are rendered if the event is fired.
+
+**Assumptions:**
+
+- This definition assumes that the event causing the changes is the only origin of changes to the [web page][] and therefore the page does not change if there is no such event. If this is not the case, changes may be attributed to the wrong event.
```

Comment:
I am still uneasy about the combination of these two.
I totally get the idea (I think) and agree with it, but the formulation seems contradictory…

It looks like we say on one hand what to do if there are other changes happening and on the other hand that no other changes are happening…

Of course, this is because the assumption is about "changes caused by something (DOM event or else in that case)" while the note above is about "ongoing changes that are not caused by anything anymore".

Given that the assumption is copied in the rules, I'd say it is the part that need clarification.

Maybe something like
> This definition/rule assumes that there are no changes in the content caused by another event.

Reply:
I see your point and I agree that it can be better formulated. I'll update based on your suggestion.